tests/foreman/cli/test_hammer.py

```diff
@@ -0,0 +1,78 @@
+"""Tests related to hammer command and its options and subcommands."""
+import json
+
+from robottelo.cli import hammer
+from robottelo.common import ssh
+from robottelo.common.helpers import read_data_file
+from robottelo.test import CLITestCase
+
+HAMMER_COMMANDS = json.loads(read_data_file('hammer_commands.json'))
```

Comment:
It will no change at runtime, this is just the mapping for the expected commands and should be updated before running the tests.